ui/org.eclipse.pde.ui.tests/src/org/eclipse/pde/ui/tests/AllPDEMinimalTests.java

```diff
@@ -49,7 +50,8 @@
 	AllPDERuntimeTests.class,
 	// ExportBundleTests.class,
 	AllLauncherTests.class, AllLogViewTests.class, ProjectCreationTests.class, BundleRootTests.class,
-	PluginRegistryTestsMinimal.class, ClasspathResolverTest.class, PDESchemaHelperTest.class,
+	PluginRegistryTestsMinimal.class, ClasspathResolverTest.class, ClasspathUpdaterTest.class,
+		PDESchemaHelperTest.class,
```

Comment:
This will not be necessary anymore once https://github.com/eclipse-pde/eclipse.pde/pull/745 is merged.